chat-plugins/trivia.js

```diff
@@ -7,13 +7,27 @@
 
 const FS = require('./../lib/fs');
 
-const CATEGORIES = {
+const MAIN_CATEGORIES = {
```

Comment:
This isn't really related to the pullreq, but this has been bothering me for years since I wrote the plugin. Wouldn't the categories be better suited as maps instead of having to override their prototypes?